macros/select_date_range.sql

```diff
@@ -0,0 +1,15 @@
+{% macro select_date_range_base(start_date, end_date) %}
+    where cast(left(replace(_table_suffix, 'intraday_', ''), 8) as int64) >= {{ start_date }}
```

Comment:
What are you trying to achieve with this line?

Reply:
This part of the code was copied from base_ga4__events.sql. If we have only set a start_date, then our table suffix date needs to be bigger then and equal to startdate, so that we get all the data from start date up untill now